tests/framework/clients/block.go

```diff
@@ -151,6 +151,11 @@ func (b *BlockOperation) ListAllImages(clusterInfo *client.ClusterInfo) ([]Block
 	// for each pool, get further details about all the images in the pool
 	images := []BlockImage{}
 	for _, p := range pools {
+		// Let's ignore to list device_health_metrics pool that sometimes waits forever.
+		// This filtering is harmless because this pool is empty in new cluster.
+		if p.Name == "device_health_metrics" {
+			break
```

Comment:
I think we want to continue with other pools and just ignore this one, right?
```suggestion
			continue
```